src/Stratis.Bitcoin.Features.Miner/Interfaces/IPosMinting.cs

```diff
@@ -29,7 +30,7 @@ public interface IPosMinting
         /// </para>
         /// </summary>
         /// <param name="walletSecret">Credentials to the wallet with which will be used for staking.</param>
-        Task GenerateBlocksAsync(PosMinting.WalletSecret walletSecret);
+        Task GenerateBlocksAsync(PosMinting.WalletSecret walletSecret, CancellationToken ct);
```

Comment:
Didn't we just agree that we will use longer names?

And again parameter comment

Reply:
But would you please stop expanding this PR?